pages/pil/application/content/index.js

```diff
@@ -16,21 +16,21 @@ module.exports = {
     }
   },
   certificate: {
-    number: 'Certificate number',
-    accreditation: 'Accreditation body',
-    awarded: 'Date awarded',
-    expiry: 'Expiry date',
-    modules: 'Modules completed',
-    exemption: 'Exempt from module',
-    reason: 'Reason for exemption'
+    number: 'Certificate number:',
+    accreditation: 'Accreditation body:',
+    awarded: 'Date awarded:',
+    expiry: 'Expiry date:',
+    modules: 'Modules completed:',
+    exemption: 'Exempt from module:',
+    reason: 'Reason for exemption:'
   },
   exemption: {
     skipped: 'No exemptions provided.'
   },
   procedure: {
-    category: 'Procedure category',
-    catDLabel: 'Evidence of competency',
-    catFLabel: 'Type of regulated procedure'
+    category: 'Procedure category:',
+    catDLabel: 'Evidence of competency:',
+    catFLabel: 'Type of regulated procedure:'
```

Comment:
We have to put the colons inside the snippet?

Reply:
yeah

Reply:
`<div><element>hi</element>:</div>` renders differently on the server and the client

Reply:
we might be able to get around it with a `<Fragment>` actually, so if you really don't like the above then this may work:

```js
<div>
  <Fragment>
    <Snippet>something</Snippet>
    :
  </Fragment>
</div>
```

Reply:
or perhaps wrap the `:` in a `<span>:</span>`

Reply:
One of these options is probably better